react/src/App.jsx

```diff
@@ -19,18 +19,6 @@ function App() {
   const [messageEventListenerIsAdded, setmessageEventListenerIsAdded] =
     useState(false);
 
-  const onUpdateHelpshiftConfig = () => {
-    window.Helpshift("updateHelpshiftConfig");
-  };
-
-  useEffect(() => {
-    const newInterval = setInterval(() => {
-      setMessage("");
-    }, 3000);
-
-    return () => clearInterval(newInterval);
-  }, [message]);
-
   const onHandleFullPrivacyChange = (e) => {
```

Comment:
@dv-Gogna The handlers should be renamed like this:
`onFullPrivacyChange`, and not `onHandleFullPrivacyChange`.

One way to think of this is - we're not calling this `onHandle` of the event. We're calling it `on<Event>`